csfieldguide/static/interactives/scene-editor/js/scene-editor.js

```diff
@@ -72,13 +114,18 @@ $(document).ready(function () {
     $("#equation-container").removeClass('d-none').addClass('col-md-8');
     $("#eqtn-title").addClass('d-none');
     $("#scene-creation-title-area").removeClass('d-none');
+    $("#delete-button").removeClass('d-none');
   }
 
   $("#selectable-objects").on('change', switchFocus);
   $("#colour-input").on('input', recolourHashBox);
 
   $("#add-object").click(newObject);
   $("#apply-transformation").click(applyTransformation);
+  $("#delete-button").click(deleteSuspect);
+
+  $('.matrix-row input').on('keyup bind cut copy paste', validateInput);
```

Comment:
This doesn't show that the input is invalid until the input box is out of focus.

I think you forgot to copy over the following css rule from `matrix-simplifier`:
````
.input-error:focus {
  border-color: #FF0000;
  box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075), 0 0 8px rgba(255, 0, 0, 0.6);
}

Reply:
I wasn't aware of it and couldn't find out how to do, thanks